pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
+
+- Display current value as octal:
+
+`umask`
+
+- Display current value symbolically:
+
+`umask -S`
+
+- Turn off permission bits with octal value 'nnnn':
```

Comment:
Ok, I'm trying to explain it. Set the mask for ... may be confusing for beginners, I googled it. A lot of people do not know what the mask is and how it works in this case.

If you insist I will change it to `- Set the mask to {{nnnn}}`